var/spack/repos/builtin/packages/fribidi/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Fribidi(AutotoolsPackage):
+    """GNU FriBidi: The Free Implementation of the Unicode Bidirectional
+    Algorithm."""
+
+    homepage = "https://github.com/fribidi/fribidi"
+    url      = "https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2"
+
+    version('1.0.5', sha256='6a64f2a687f5c4f203a46fa659f43dd43d1f8b845df8d723107e8a7e6158e4ce')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    # TODO: If https://github.com/spack/spack/pull/12344 is merged, this
```

Comment:
@goxberry now that #12344 has been merged, you can remove this

Reply:
@scheibelp Done in https://github.com/spack/spack/commit/2ae0268043b257cf6e9bc02e9ff145da5cdca453, and rebased on `develop`.